docs/design/fabric-v2+/fpc-validation.puml

```diff
@@ -52,88 +52,107 @@ group endorsement
     end ref
     return proposal response
 
+    alt FPC "light"
+        User1 -> ECC1 ++: invoke "validateEnclaveEndorsement" (proposal response)
+            group enclave endorsement validation
+                ECC1 -> ECC1 : <args, read/writeset, result> <- extract from transaction
+                ECC1 -> ECC1 : <Enclave_VK, Sig_Enclave, ECC_result> <- extract from result
+
+                ECC1 -> ECC1 : verify Sig_Enclave over <args, read/writeset, ECC_result> with Enclave_VK
+                ECC1 -> ECC1 : Enclave_Id <- Hash(Enclave_VK)
+                create participant ERCC order 30
+                ECC1 -> ERCC++ : queryEnclaveCredentials(CC_Id, Enclave_Id)
+                    note right
+                        If FPC enclave is registered, ERCC will
+                        return the corresponding Credentials,
+                        otherwise Credentials is empty.
+                    end note
+                return Credentials
```

Comment:
here some minor changes are needed. From `ccresp_msg` we already get `Enclave_Id`, so no need to hash `Enclave_VK`. In particular, we also first need to fetch `Enclave_VK` from ERCC before we can verify the signature.

Reply:
good find, order of the two steps has to be swapped and no hash computation needed